src/components/action-popover/action-popover.spec.js

```diff
@@ -259,18 +259,19 @@ describe("ActionPopover", () => {
         wrapper
           .find(MenuButton)
           .props()
-          .onClick({ stopPropagation: () => {} });
+          .onClick({ stopPropagation: () => {}, preventDefault: () => {} });
       });
 
       act(() => {
         wrapper.update();
       });
 
       act(() => {
-        wrapper
-          .find(StyledMenuItem)
-          .props()
-          .onKeyDown({ which: 13, stopPropagation: jest.fn() });
+        wrapper.find(StyledMenuItem).props().onKeyDown({
+          which: 13,
+          stopPropagation: jest.fn(),
+          preventDefault: jest.fn(),
```

Comment:
No need to make these `jest.fn()` if we're not checking they've been called anywhere
```suggestion
          preventDefault: () => {},
```